src/main/webapp/resources/js/components/icons/Icons.jsx

```diff
@@ -230,8 +231,9 @@ export const IconCalendarTwoTone = ({ ...props }) => (
 
 export const IconShare = ({ ...props }) => <ShareAltOutlined {...props} />;
 
-export const IconWarningOutlined = ({ ...props }) =>
-  <WarningOutlined {...props} />;
+export const IconWarningOutlined = ({ ...props }) => (
+  <WarningOutlined {...props} />
```

Comment:
It's fine if you want to add this. Eric and I actually had a conversation recently, and we are going to get ride of this file.  It is redundant.  If you get rid of Ant Design we have a lot more to do than change icons.

Reply:
Removed the icon I added and made use of the icon directly from the ant design icons package in 66fd23c47ca7b634ae1918ac77341a2fdf53048e